### PR TITLE
Added a draft version of minnie-kenny.sh BA-5810

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: src/ci/bin/test.inc.sh [hunk 1]
```diff
@@ -633,6 +637,17 @@ cromwell::private::install_wait_for_it() {
     chmod +x "$CROMWELL_BUILD_WAIT_FOR_IT_SCRIPT"
 }
 
+cromwell::private::install_git_secrets() {
```
Comment (cjllanwarne) [inline src/ci/bin/test.inc.sh]: If we already have this, do we need the `-f` in the minnie_kenny script?

File: minnie-kenny.sh [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        ;;
+      *)
+        echo_err "Unknown argument: $1"
+        usage
+        ;;
+    esac
+  done
+
+  if [ "${minnie_kenny_gitconfig}" = "" ]; then
+    echo_err "Error: you need to provide an include file."
```
[the remark this answers is not in the document]
Comment (kshakir) [reply minnie-kenny.sh]: Put something similar.

File: minnie-kenny.sh [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+    else
+      echo_err "Error: git-secrets is not installed into the expected git hooks" \
+        "'commit-msg' 'pre-commit' and 'prepare-commit-msg'."
+      error_modify=1
+    fi
+  elif [ ${actual} -ne ${expected} ]; then
+    echo_err "Error: git-secrets is not installed into all of the expected git hooks." \
+      "Double check the 'commit-msg' 'pre-commit' and 'prepare-commit-msg' under the directory" \
+      "${minnie_kenny_git_dir}/hooks and consider running \`git secrets --install --force\`."
+    exit 1
```
[the remark this answers is not in the document]
Comment (kshakir) [reply minnie-kenny.sh]: Refactored just a bit. Hopefully flows a little bit easier.

File: project/ContinuousIntegration.scala [hunk 4]
```diff
@@ -20,6 +21,7 @@ object ContinuousIntegration {
       IO.copyDirectory(srcCiResources.value, targetCiResources.value)
     },
     renderCiResources := {
+      minnieKenny.toTask(" -f").value
```
[the remark this answers is not in the document]
Comment (kshakir) [reply project/ContinuousIntegration.scala]: No longer running `minnie-kenny.sh` with `-f` during `sbt renderCiResources`.

File: src/ci/bin/test.inc.sh (hunk 1 above)
Comment (kshakir) [reply src/ci/bin/test.inc.sh]: Chatted IRL. This section of bash-code only installs the `git-secrets` executable into the `$PATH`. It does not add git-secrets to the git hooks, no use the secrets definitions in minnie-kenny.gitconfig. Those are handled elsewhere by running `minnie-kenny.sh -f`.

**Pull request merged**